.rubocop_todo.yml

```diff
@@ -1,62 +1,44 @@
 # This configuration was generated by
 # `rubocop --auto-gen-config`
-# on 2020-05-15 17:53:23 +0000 using RuboCop version 0.52.1.
+# on 2022-12-24 00:49:23 +0000 using RuboCop version 0.52.1.
```

Comment:
why are we updating the todo? I'm concerned that we just decided to ignore problems instead of fixing them?